src/components/modals/CreateAntModal/CreateAntModal.tsx

```diff
@@ -0,0 +1,594 @@
+                                              : 'transparent',
+                                          border: 'none',
+                                          borderRadius: '2px',
+                                          color:
+                                            editingField === row.attribute
+                                              ? 'black'
+                                              : 'white',
+                                          padding:
+                                            editingField === row.attribute
+                                              ? '10px '
+                                              : '10px 0px',
+                                          display: 'block',
+                                        }}
+                                        disabled={
+                                          editingField !== row.attribute
+                                        }
+                                        placeholder={`Enter a ${mapKeyToAttribute(
+                                          row.attribute,
+                                        )}`}
+                                        value={
+                                          editingField !== row.attribute
+                                            ? row.value
+                                            : modifiedValue
+                                        }
+                                        setValue={(e) => {
+                                          setModifiedValue(e);
+                                          row.value = e;
+                                        }}
+                                        validationPredicates={
+                                          row.attribute === 'owner' ||
+                                          row.attribute === 'controller' ||
+                                          row.attribute === 'targetID'
+                                            ? {
+                                                [VALIDATION_INPUT_TYPES.ARWEAVE_ID]:
+                                                  (id: string) =>
+                                                    arweaveDataProvider.validateArweaveId(
+                                                      id,
+                                                    ),
+                                              }
+                                            : {
+                                                [VALIDATION_INPUT_TYPES.UNDERNAME]:
+                                                  (value: string) => {
+                                                    return new Promise(
+                                                      (resolve) => {
+                                                        resolve(value);
+                                                      },
+                                                    );
+                                                  },
```

Comment:
what is this doing?

Reply:
placeholder validation. Validation input complained about not having one - we also havent set up rules for undernames so this should actually be updated with that when we know.

Reply:
lets remove it if it's not doing anything and put a TODO

Reply:
okey dokey, removed